examples/kt_jvm_export/BUILD

```diff
@@ -0,0 +1,19 @@
+load("@rules_jvm_external//:defs.bzl", "kt_jvm_export")
+
+# To export the file, run:
+#
+# `bazel run //:example-export.publish --define "maven_repo=file://$(pwd)/repository"`
+#
+# GPG signing will be enabled if you also define `gpg_sign=true`, `maven_user`, and
+# `maven_password`
+
+kt_jvm_export(
+    name = "example-export",
```

Comment:
Can you add a test to verify that the contents of this exported jar is what is expected?

Reply:
Just added tests in the integration folder near the tests for java_export.